DockeriseRmap/description.txt

```diff
@@ -0,0 +1,27 @@
+Run with:
+docker run --rm -v /workspaces/ExpansionHunter/results:/results:ro -v /workspaces/ExpansionHunter/results:/results2:rw -ti <Image ID>
+
+where results is the location of the current data (will be mentioned in A, C, D and E)
+
+Then:
+cd ../..
+bgzip -c results/repeats.vcf > workdir/repeats.vcf.gz
+tabix -p vcf workdir/repeats.vcf.gz
```

Comment:
Does the file need to be indexed?  I would have expected the VCF to be sequentially processed and appers to be read with a simple `read_table`, plus it appears to be the uncompressed and unindexed file passed to the R script

Reply:
It is used in a bcftools command and the .gz suffix is added in the script